examples/Table-nested-items.rb

```diff
@@ -27,6 +29,8 @@ def main_dialog
             VSpacing(0.2),
             Left(Label("Selected:")),
             Label(Id(:selected), Opt(:outputField, :hstretch), "..."),
+            Left(Label("Open Items:")),
+            Label(Id(:open_items), Opt(:outputField, :hstretch), "..."),
```

Comment:
Nice, this makes screen testing possible without grepping y2log (like with some other UI examples)

Can you please put these 2 in a HBox so that I can grep the screen dump for "OpenItems:.*certain_id"? Likewise with "Selected"

Reply:
In general, we should not need to rely on grepping the y2log for tests; that would result in a very fragile test environment. I'd very much prefer to adapt the examples used for testing to make everything relevant available on the screen; that shouldn't be too hard in general, and it would also improve the usability of those examples.

Previously, one aim of the simple examples was to showcase the simplest possible use case for a widget. But over time it turned out that app developers seldom go beyond the first example, so I am not trying to showcase the _normal_ use case, not the simplest one; even if that means that the example becomes somewhat more complex.

If we are indeed adding more automated tests, we should really go that route and add more advanced examples. I think the investment would be worthwhile.

Reply:
From IRC:

> <mvidner> HuHa: if we put "sda" on its own line, "grep" cannot distinguish it from the live table contents. "grep 'Open Items:.*sda '"  can
>
> <mvidner> HuHa: just put it on a single line

Reply:
ok

Reply:
Done.